modules/nw-osp-pod-creating-ipv6.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-pod-creating-ipv6_{context}"]
+= Create pods that have IPv6 connectivity on {rh-openstack}
+
+After you enable IPv6 connectivty for pods and add it to them, create pods that have secondary IPv6 connections. 
+
+.Procedure
+
+. Define a `Pod` object that uses your IPv6 namespace and the annotation `k8s.v1.cni.cncf.io/networks: <additional_network_name>`, where `<additional_network_name` is the name of the additional network.
```

Comment:
@itzikb-redhat do you have an example handy?

Reply:
@MaysaMacedo Itzik offered this:

```suggestion
. Define a `Pod` object that uses your IPv6 namespace and the annotation `k8s.v1.cni.cncf.io/networks: <additional_network_name>`, where `<additional_network_name` is the name of the additional network. For example:
+
[source,yaml]
----
apiVersion: apps/v1
kind: Deployment
metadata:
  name: hello-openshift
spec:
  affinity:
    podAntiAffinity: 
      requiredDuringSchedulingIgnoredDuringExecution:
         - labelSelector:
            matchExpressions:
            - key: app
              operator: In 
              values:
              - hello-openshift
  replicas: 2
  selector:
    matchLabels:
      app: hello-openshift
  template:
    metadata:
      labels:
        app: hello-openshift
      annotations:
        k8s.v1.cni.cncf.io/networks: hostonly-v6
    spec:
      securityContext:
        runAsNonRoot: true
        seccompProfile:
          type: RuntimeDefault
      containers:
      - name: hello-openshift
        securityContext:
          allowPrivilegeEscalation: false
          capabilities:
            drop:
            - ALL
        image: quay.io/openshift/origin-hello-openshift
        ports:
        - containerPort: 8080
----
```

I'd have to change the language around a bit, as it's a deployment rather than just a pod, I assume, but does that otherwise seem in line to you?

Reply:
Looks ok to me.